index.js

```diff
@@ -2,22 +2,67 @@ import 'react-native-get-random-values'; // must import before @ethersproject/sh
 import '@ethersproject/shims';
 import './shim';
 import '@walletconnect/react-native-compat';
-import {AppRegistry} from 'react-native';
+import {AppRegistry, Alert} from 'react-native';
+import 'react-native-get-random-values';
```

Comment:
Why is this import needed?

Reply:
Oh... it's duplicated (from line 1). Good catch!